pkg/agent/plugin/workloadattestor/docker/docker.go

```diff
@@ -74,7 +74,7 @@ type dockerPluginConfig struct {
 
 	UnusedKeyPositions map[string][]token.Pos `hcl:",unusedKeyPositions"`
 
-	Experimental experimentalConfig `hcl:"experimental,omitempty" json:"experimental,omitempty"`
+	Experimental experimentalConfig `hcl:"experimental,omitempty" json:"experimental"`
```

Comment:
I think modernize removed `omitempty` here because that doesn't actually do anything for fields of struct type (it can never be "empty"). Possibly what was actually meant was `omitzero`, in which case the field would be omitted if the struct has the "zero" value; but that would be a change in behaviour.

Likely the same applies for the `hcl` annotation, but modernize didn't touch that so neither did I.